packages/docs/developer-resources/walkthroughs/hellocelo.md

```diff
@@ -57,76 +61,84 @@ At any point in the file you can `console.log()` variables to print their output
+{% endhint %}
 
-Reading other account balances is interesting, but how can we send value to each other on the test net?
+Reading all account balances is a powerful feature of blockchains. Next, let's see how we can send value to each other on the test net.
 
-We need to do a few things:
+In order to do transfers (aka [transactions](https://docs.celo.org/getting-started/glossary#transaction)), we need to:
 
-1. Create an account \(create a private key\)
+1. Create an [account](https://docs.celo.org/getting-started/glossary#account) \(by creating a private key\)
 2. Fund it with test cGLD and cUSDs
 3. Sign and send transactions to the network
 
 ## Accounts
 
-We are accessing the Celo network via a remote node via HTTP requests at `'https://alfajores-forno.celo-testnet.org'`.
+We are accessing the Celo network via a remote [node](https://docs.celo.org/getting-started/glossary#node) via HTTP requests at `'https://alfajores-forno.celo-testnet.org'`.
 
+{% hint style="info" %}
 Don't worry about what this means right now, just understand that it is easier to get started using Celo by accessing remote nodes, rather than running them locally on your machine. You can [read more about the details of the Celo network here.](https://github.com/critesjosh/celo-monorepo/tree/8542c1bc3ad32bc48eed33073f4d34a36fd91fae/packages/docs/celo-sdk/walkthroughs/overview.md#topology-of-a-celo-network)
+{% endhint %}
 
 Because we are accessing the network remotely, we need to generate an account to sign transactions and fund that account with test cGLD.
 
-There is a short script in `getAccount.js` to either get a Celo account from a mnemonic in the `.secret` file, or create a random account if the file is empty. In the script, we use`web3.js` to create a new private key/account pair. [Web3.js](https://web3js.readthedocs.io/en/v1.2.6/) is a popular javascript library for handling Ethereum related functionality. Celo is a cousin of Ethereum, so this library will work well for generating new Celo accounts.
+There is a short script in `getAccount.js` to either get a Celo account from a mnemonic in the `.secret` file, or create a random account if the file is empty. In the script, we use`web3.js` to create a new private key/account pair. [Web3.js](https://web3js.readthedocs.io/en/v1.2.6/) is a popular javascript library for handling Ethereum related functionality. Celo is a cousin of Ethereum, so this library works well for generating Celo accounts.
 
 {% hint style="danger" %}
 This is not the standard way of managing Celo accounts. In a production environment, the [Celo Wallet](../../celo-codebase/wallet/) will manage accounts for you. Accessing accounts from the Celo Wallet will be discussed in future guides.
```

Comment:
I might even go a step further here and recommend using a hardware wallet or similar utility to generate a private/public key pair